src/ObjLoading/Parsing/LocalizeFile/Sequence/SequenceLocalizeFileLanguageValue.cpp

```diff
@@ -31,5 +31,13 @@ void SequenceLocalizeFileLanguageValue::ProcessMatch(LocalizeFileParserState* st
     state->m_current_reference_languages.emplace(langName);
 
     if (langName == state->m_language_name_caps)
-        state->m_entries.emplace_back(state->m_current_reference, valueToken.StringValue());
+    {
+        const auto& currentReference = state->m_current_reference;
+        if (const auto i = state->m_entries.find(currentReference); i != state->m_entries.end())
+        {
+            std::cout << "Localize: Reference \"" << currentReference << "\" was already defined. Using new value \"" << valueToken.StringValue() << "\"\n";
```

Comment:
I think logging the value is a bit too much :sweat_smile: Probably too verbose since it can also be multi-line and everything.
The main point is to inform the user that there is a duplicate key so i guess just including the reference should be good enough :stuck_out_tongue: